lib/Test/MockFile.pm

```diff
@@ -1227,7 +1254,10 @@ BEGIN {
     };
 
     *CORE::GLOBAL::opendir = sub(*$) {
-        @_ = _upgrade_barewords(@_) if defined $_[0];
+        if ( defined $_[0] ) {
+            @_ = _upgrade_barewords(@_) if defined $_[0];
```

Comment:
useless postfix check here `... if defined $_[0]` it was just checked above

Reply:
Yeah, that's a good point. I moved things around a lot and missed it. Just force-pushed a fixed commit.